Sources/SwiftFusion/Core/FixedShapeTensor.swift

```diff
@@ -70,6 +70,10 @@ extension FixedShapeTensor {
   }
 
   public static var zero: Self { .init(Tensor(zeros: Self.shape)) }
+
+  public var zeroTangentVectorInitializer: () -> Self {
```

Comment:
Why is this part of the "test vector semantics" change?  Is it a bug fix?

Reply:
hmm, this doesn't look like it's necessary and the tests still pass when I remove it. so I'll remove it.